charts/spire/charts/spire-server/templates/NOTES.txt

```diff
@@ -4,3 +4,15 @@ Installed {{ .Chart.Name }}…
 
   kubectl exec -n {{ .Release.Namespace }} {{ include "spire-server.fullname" . }}-0 -c spire-server -- \
     spire-server entry show
+
+{{- if eq (.Values.tornjak.enabled | toString) "true" }}
+
```

Comment:
Hmm... Without a networkpolicy or some kind of user auth, anyone within the cluster can poke at things? Should we put a hard to miss  warning here?

Reply:
Since this is just for a demo purpose running locally on *kind* or *minikube*, should we really be concern? 
In production we would be using Ingress and User Management that controls the Tornjak API access.

Reply:
The feature is at demo level, but people might start utilizing it without realizing that? We want to make sure they know that is the case.

Reply:
OK, I will put some Warning message on the screen via NOTES